test/roby_app/test_configuration.rb

```diff
@@ -65,4 +66,111 @@ def stub_deployment(name)
             assert_equal @task_m.orogen_model, deployed_task.task_model
         end
     end
+
+    describe "#connect_to_orocos_process_server" do
+        before do
+            @conf = Syskit::RobyApp::Configuration.new(Roby.app)
+            @mock_process = flexmock(pid: 20)
+            @mock_process.should_receive(:spawn).by_default
+            @mock_process.should_receive(:kill).by_default
+            @mock_process.should_receive(:dead!).by_default
+            @available_project_names = []
+            @available_typekit_names = []
+            @available_deployment_names = ['deployment']
+
+            set_log_level ::Robot, Logger::FATAL + 1
+            set_log_level Orocos::RemoteProcesses::Server, Logger::FATAL + 1
```

Comment:
Are these hacks to disable logs?

Reply:
It's to temporarily set the logger levels to avoid having them output stuff during the tests. They're reset on teardown.